Gruntfile.coffee

```diff
@@ -144,28 +144,28 @@ module.exports = (grunt) ->
           modules: [{
             name: 'main'
             include: [
+              'settings/base'
+              'settings/config'
               'cs!pages/error/error'
               'cs!pages/home/home'
               'cs!pages/contents/contents'
               'cs!pages/search/search'
               'cs!pages/browse-content/browse-content'
-              'cs!pages/workspace/workspace'
               'cs!pages/about/about'
               'cs!pages/tos/tos'
               'cs!pages/license/license'
               'cs!pages/donate/donate'
-              'cs!pages/role-acceptance/role-acceptance'
+              'cs!helpers/backbone/views/editable'
 
-              # FIX: edit modules should be loaded in separate modules
               'select2'
               'bootstrapPopover'
+              'cs!pages/workspace/workspace'
+              'cs!pages/role-acceptance/role-acceptance'
               'cs!modules/media/editbar/editbar'
-              'cs!helpers/backbone/views/editable'
-
               'cs!configs/aloha'
             ]
             exclude: ['coffee-script', 'less/normalize']
-            excludeShallow: ['settings']
+            excludeShallow: ['settings', 'OpenStaxConceptCoach']
```

Comment:
Just so I understand, this PR will only load Concept Coach when it is needed, correct? So the home page will not be downloading Concept Coach code and css.